File: Ambev.DeveloperEvaluation.Shared/Validations/ValidationMessages.cs
namespace Ambev.DeveloperEvaluation.Shared.Validations;

public static class ValidationMessages
{
    public const string MustAccept = "É necessário aceitar as condições";
    public const string FieldRequired = "{field} Requerido";
    public const string FieldValueInvalid = "{field} Inválido";
    public const string MaxChars = "Máximo de {1} caracter(es) permitido(s)";
    public const string MinChars = "Mínimo de {1} caracter(es) necessário(s)";
    public const string NeededChars = "{1} carater(es) necessário(s)";
    public const string PasswordsNoMatch = "As senhas não conferem";
    public const string InvalidCaptcha = "Captcha incorreto";
    public const string NotValidableCaptcha = "Captcha não validável";
    public const string InvalidPhone = "O telefone incluindo DDD deve ter 10 números";
    public const string InvalidMobile = "O celular incluindo DDD deve ter 11 números";
    public const string DefaultAuthenticationError = "Usuário ou senha inválida";
    public const string UserLockedOut = "Usuário bloqueado. Tente novamente dentro de alguns minutos";
    public const string IsNotAllowed = "Usuário bloqueado. Tente novamente dentro de alguns minutos";
}

File: ambev.developerevaluation.client/src/utils/validation-messages.ts
export class ValidationMessages {
  public static readonly MustAccept: string =
    "É necessário aceitar as condições"
  public static readonly FieldRequired: string = "{field} é obrigatório(a)"
  public static readonly FieldValueInvalid: string = "{field} inválido(a)"
  public static readonly MaxChars: string =
    "Máximo de {1} caracter(es) permitido(s)"
  public static readonly MinChars: string =
    "Mínimo de {1} caracter(es) necessário(s)"
  public static readonly NeededChars: string = "{1} carater(es) necessário(s)"
  public static readonly PasswordsNoMatch: string = "As senhas não conferem"
  public static readonly DefaultAuthenticationError: string =
    "Usuário ou senha inválida"
  public static readonly UserLockedOut: string =
    "Usuário bloqueado. Tente novamente dentro de alguns minutos"
  public static readonly IsNotAllowed: string =
    "Usuário bloqueado. Tente novamente dentro de alguns minutos"
}
